pkg/controller/replication_controller.go

```diff
@@ -72,6 +74,20 @@ func (r RealPodControl) createReplica(namespace string, controller api.Replicati
 		desiredAnnotations[k] = v
 	}
 
+	createdByRef, err := api.GetReference(&controller)
+	if err != nil {
+		util.HandleError(fmt.Errorf("unable to get controller reference: %v", err))
+		return
+	}
+
+	createdByRefJson, err := json.Marshal(createdByRef)
```

Comment:
It's a bit awkward, because the field doesn't go through conversion, so the versioned rep isn't really appropriate, either. I do want to get rid of the json tags on the internal rep., however, which would also break this.